src/Campaigns/Migrations/MigrateFormsToCampaignForms.php

```diff
@@ -77,17 +77,43 @@ protected function getFormData(): array
         })
             ->whereIsNull('campaigns.id');
 
-        // Exclude forms with an `upgraded` status, which are archived.
-        $query->where('forms.post_status', 'upgraded', '!=');
+        /**
+         * Exclude forms with an `upgraded` status, which are WP revisions.
+         *
+         * @see https://wordpress.org/documentation/article/post-status/#auto-draft
+         */
+        $query->where('forms.post_status', 'auto-draft', '!=');
```

Comment:
I think this is conflating two different things. We do need to exclude `auto-draft` as a status, but I don't think that relates in any way to a form being "upgraded" (as stated in the comment above).

Reply:
Resolved in f40ab332e726bc2de3f7104e59a001a506099864